rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc

```diff
@@ -21,7 +21,7 @@ User API keys are required for the integration of RPA processes in a MuleSoft Co
 
 |===
 
-* The user who will use an API key must exist in the *Users* view of the *User Management* module.
+* The user who use an API key must exist in the *Users* view of the *User Management* module.
```

Comment:
```suggestion
* The user who uses an API key must exist in the *Users* view of the *User Management* module.
```

Reply:
Maybe add a comment to Vale to check for correct conjugation in case of future tense replacements :)